From 707442d1e89601ddd675518e6aa2e4ee79d1c8b0 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Mon, 5 Oct 2020 21:02:11 -0700
Subject: [PATCH 09/15] Deprioritize important developer notifications

Informing the user that ADB is enabled does not necessitate an alert
sound. It's especially annoying with the default AOSP notification sound,
but even with a less distracting sound, this doesn't need to be so
intrusive.

Change-Id: Idee92b44cff2473a3686a2f2d95c6f5e4e91c2f8
Reviewed-on: https://review.statixos.com/c/android_frameworks_base/+/5668
Reviewed-by: Sourajit Karmakar <sourajit@live.com>
Tested-by: Sourajit Karmakar <sourajit@live.com>
Reviewed-on: https://review.statixos.com/c/android_frameworks_base/+/7146
Reviewed-by: Anay Wadhera <anay1018@gmail.com>
Tested-by: Anay Wadhera <anay1018@gmail.com>
Reviewed-on: https://review.statixos.com/c/android_frameworks_base/+/8667
Reviewed-by: Sourajit Karmakar <sourajitk@statixos.com>
Tested-by: Sourajit Karmakar <sourajitk@statixos.com>
Signed-off-by: Anushek Prasal <anushekprasal@gmail.com>
---
 .../internal/notification/SystemNotificationChannels.java       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/notification/SystemNotificationChannels.java b/core/java/com/android/internal/notification/SystemNotificationChannels.java
index 681b46a01c8d..89e6d1f50307 100644
--- a/core/java/com/android/internal/notification/SystemNotificationChannels.java
+++ b/core/java/com/android/internal/notification/SystemNotificationChannels.java
@@ -113,7 +113,7 @@ public class SystemNotificationChannels {
         final NotificationChannel developerImportant = new NotificationChannel(
                 DEVELOPER_IMPORTANT,
                 context.getString(R.string.notification_channel_developer_important),
-                NotificationManager.IMPORTANCE_HIGH);
+                NotificationManager.IMPORTANCE_MIN);
         developer.setBlockable(true);
         channelsList.add(developerImportant);
 
-- 
2.25.1

